model/pack/QtiItemPacker.php

```diff
@@ -127,12 +129,40 @@ public function packQtiItem($item, $lang, $qtiItem, Directory $directory)
         return $itemPack;
     }
 
+    /**
+     * @param $item
+     * @param $qtiItem
+     * @param Directory $directory
+     * @param PackedAsset[] $packedAssets
+     * @return ItemPack
+     * @throws common_Exception
+     */
+    public function createQtiItemPackWithAssets($item, $qtiItem, Directory $directory, array $packedAssets): ItemPack
+    {
+        try {
+            $itemPack = new ItemPack(self::$itemType, $qtiItem->toArray());
+            $itemPack->setAssetEncoders($this->getAssetEncoders());
+
+            /** @var PackedAsset $packedAsset */
+            foreach ($packedAssets as $packedAsset) {
+                if ($packedAsset->getType() != 'xinclude') {
+                    $itemPack->setAsset($packedAsset->getType(), $packedAsset->getMediaAsset(), $directory);
```

Comment:
Is it still the 3rd removed argument ?  https://github.com/oat-sa/extension-tao-item/pull/410#diff-ffe2f6ab88514940d57e22b360f6fe2fR139

Reply:
good catch, it is the purpose of other PR 😞